datafusion/sqllogictest/test_files/unnest.slt

```diff
@@ -165,6 +165,34 @@ select unnest(column1), column1 from unnest_table;
 6 [6]
 12 [12]
 
```

Comment:
Are there any `.slt` tests for unnesting structures? these tests seem to be on arrays (Lists) rather than Structs